pkg/goldpinger/pod_selecter.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Bloomberg Finance L.P.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package goldpinger
+
+import (
+	"github.com/cespare/xxhash"
+	rendezvous "github.com/dgryski/go-rendezvous"
```

Comment:
Travis picked up that you forked it to `stuartnelson3` namespace, but didn't update the `Gopkg.toml`. Probably ran on your machine because you had both in your `GOPATH`.

Reply:
thanks for catching that!